.docker/start.dev.sh

```
@@ -0,0 +1,5 @@
+#/bin/bash
```

Shellcheck (0.10.0)

[error] Use #!, not just #, for the shebang.

(SC1113)